packages/compiler/test/core/compiler-code-fixes/triple-quote-indent.codefix.test.ts

```diff
@@ -0,0 +1,77 @@
+import { strictEqual } from "assert";
+import { describe, it } from "vitest";
+import { createTripleQuoteIndentCodeFix } from "../../../src/core/compiler-code-fixes/triple-quote-indent.codefix.js";
+import { SyntaxKind } from "../../../src/index.js";
+import { expectCodeFixOnAst } from "../../../src/testing/code-fix-testing.js";
+
+describe("CodeFix: triple-quote-indent", () => {
+  it("case 1: each triple-quote is on a new line", async () => {
+    await expectCodeFixOnAst(
+      `
+      const a = ┆"""\r\none\r\n  two\r\n  """;
```

Comment:
please don't use `\r\n` in test, can this not be just done with actual new lines, thjis is hard to read